python/cudf/cudf/_lib/column.pyx

```diff
@@ -52,6 +53,19 @@ from cudf._lib.cpp.scalar.scalar cimport scalar
 from cudf._lib.scalar cimport DeviceScalar
 
 
+cdef get_element(column_view col_view, size_type index):
```

Comment:
I briefly thought this was identical to `copying.pyx:get_element` ,but that takes a `cudf.Column` :(

Reply:
Yes. That's why I couldn't reuse it.